client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverImpersonateUser.java

```diff
@@ -47,17 +50,21 @@
 
     @BeforeClass
     public void setup()
+            throws IOException
```

Comment:
just `throws Exception`

Reply:
I don't like throwing general exception. I know this is fine in test but I still prefer to use a specific exceptions unless there is more than one.